Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -10,7 +10,7 @@
         @paginationFragment
     }
     <Table @ref="@tableRef" ElementId="@ElementId" Class="@ClassNames" Style="@Style" Margin="@Margin" Padding="@Padding" Striped="@Striped" Bordered="@Bordered" Borderless="@Borderless" Hoverable="@Hoverable" Narrow="@Narrow" Responsive="@Responsive" Resizable="@Resizable" ResizeMode="@ResizeMode" FixedHeader="@(Virtualize || FixedHeader)" FixedHeaderTableHeight="@GetFixedTableHeaderHeight()">
-        <TableHeader>
+        <TableHeader ThemeContrast="HeaderThemeContrast">

```

Comment:
Just add `@` in front of the variable. Always prefer it that way `ThemeContrast="@HeaderThemeContrast"`.

Reply:
Done